pubspec.yaml

```diff
@@ -29,6 +29,8 @@ dependencies:
   google_fonts: ^6.2.1
   google_maps_flutter: ^2.6.1
   flutter_svg: ^2.0.10+1
+  timeago: ^3.6.1
```

Comment:
I think that we should explicitly mention any change to `pubspec.yaml` in the PR description to make sure we don't overlook something critical here.

Reply:
Thanks, I updated the description.